_includes/related_posts.html

```diff
@@ -51,6 +58,13 @@
 					{% assign author = relevantPostAttributes[5] %}
 					{% assign postContent = relevantPostAttributes[6] %}
 					{% assign postDate = relevantPostAttributes[7] %}
+					{% assign abstract = relevantPostAttributes[8] %}
+					
+					{% if abstract %}
```

Comment:
@Gryff good eye. it does not fail, but abstract is null/empty and does not pass the check on line 63: ``if abstract``.